scipy/sparse/linalg/dsolve/setup.py

```diff
@@ -28,8 +28,13 @@ def configuration(parent_package='',top_path=None):
     sources = list(glob.glob(join(superlu_src, '*.c')))
     headers = list(glob.glob(join(superlu_src, '*.h')))
     if os.name == 'nt' and ('FPATH' in os.environ or 'MKLROOT' in os.environ):
```

Comment:
The whole if statement should be removed --- there's no lsame.c in the superlu sources any more.